evidence/kishansagathiya/0001-kishansagathiya-2024-H1.md

```diff
@@ -0,0 +1,66 @@
+# Evidence-0001: Rentention at Rank I
+
+|                 |                                 |
+| --------------- |---------------------------------|
+| **Report Date** | Date of Submission (2024/07/06) |
+| **Submitted by**| Kishan Sagathiya                |
+
+
+## Member details
+
+- Matrix username: @kishanmsagathiya:matrix.org
+- Polkadot address: 13DWAWRTVpkPwWdFDmtUfh73KeCWMiJqEV84xRwpg34EZt8Y
+- Current rank: 1
+- Date of initial induction:
```

Comment:
kishan's initial [induction date](https://github.com/polkadot-fellows/seeding/pull/31) was "seeding".

Reply:
@kishansagathiya, I don't see you inducted to the Core Fellowship.
This means that your evidence is not yet required at this point in time.